website/docs/error-reference.md

```diff
@@ -1418,6 +1649,47 @@ MyMap[String, String].new
 MyMap[String, T.untyped].new
 ```
 
+## 7011
+
+Historically, users seeing this error has represented finding a bug in Sorbet
+(or at least finding a test case for which there could be a better error
+message). Consider searching for similar bugs at
+<https://github.com/sorbet/sorbet/issues> or reporting a new one.
+
+## 7013
+
+Sorbet detected that an instance variable was reassigned with different types:
+
+```rb
+class A
+  extend T::Sig
+
+  sig { void }
+  def initialize
+    @x = T.let(0, Integer)
+  end
+
+  sig { void }
+  def foo
+    @x = 'not an integer' # error: Expected `Integer` but found `String("not an integer")` for field
+  end
+end
+```
+
+If the instance variable can hold both an `Integer` and a `String`, maybe the
+type specified with `T.let` should be enlarged:
+
+```ruby
+@x = T.let(0, Object)
+```
```

Comment:
Any reason to not use `T.any(Integer, String)` here?  (This is a minor point and not blocking to landing this.)